tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1

```diff
@@ -0,0 +1,264 @@
+    [String] $Location,
+    [Switch] $UseK8s,
+    [string] $Tag
+)
+#Requires -RunAsAdministrator
+New-Variable -Name gAksEdgeQuickStartVersion -Value "1.0.231016.1400" -Option Constant -ErrorAction SilentlyContinue
+
+New-Variable -Option Constant -ErrorAction SilentlyContinue -Name arcLocations -Value @(
+    "westeurope", "eastus", "westcentralus", "southcentralus", "southeastasia", "uksouth",
+    "eastus2", "westus2", "australiaeast", "northeurope", "francecentral", "centralus",
+    "westus", "northcentralus", "koreacentral", "japaneast", "eastasia", "westus3",
+    "canadacentral", "eastus2euap"
+)
+
+if (! [Environment]::Is64BitProcess) {
+    Write-Host "Error: Run this in 64bit Powershell session" -ForegroundColor Red
+    exit -1
+}
+#Validate inputs
+$skipAzureArc = $false
+if ($arcLocations -inotcontains $Location) {
+    Write-Host "Error: Location $Location is not supported for Azure Arc" -ForegroundColor Red
+    Write-Host "Supported Locations : $arcLocations"
+    exit -1
+}
+
+$installDir = $((Get-Location).Path)
+$productName = "AKS Edge Essentials - K3s"
+$networkplugin = "flannel"
+if ($UseK8s) {
+    $productName ="AKS Edge Essentials - K8s"
+    $networkplugin = "calico"
+}
+
+# Here string for the json content
+$aideuserConfig = @"
+{
+    "SchemaVersion": "1.1",
+    "Version": "1.0",
+    "AksEdgeProduct": "$productName",
+    "AksEdgeProductUrl": "",
+    "Azure": {
+        "SubscriptionName": "",
+        "SubscriptionId": "$SubscriptionId",
+        "TenantId": "$TenantId",
+        "ResourceGroupName": "angop-test-aksedge-rg",
+        "ServicePrincipalName": "angop-test-aksedge-sp",
+        "Location": "$Location",
+        "CustomLocationOID":"",
+        "Auth":{
+            "ServicePrincipalId":"",
+            "Password":""
+        }
+    },
+    "AksEdgeConfigFile": "aksedge-config.json"
+}
+"@
+$aksedgeConfig = @"
+{
+    "SchemaVersion": "1.9",
+    "Version": "1.0",
+    "DeploymentType": "SingleMachineCluster",
+    "Init": {
+        "ServiceIPRangeSize": 10
+    },
+    "Network": {
+        "NetworkPlugin": "$networkplugin",
+        "InternetDisabled": false
+    },
+    "User": {
+        "AcceptEula": true,
+        "AcceptOptionalTelemetry": true
+    },
+    "Machines": [
+        {
+            "LinuxNode": {
+                "CpuCount": 8,
+                "MemoryInMB": 8192,
+                "DataSizeInGB": 40,
+                "LogSizeInGB": 4
+            }
+        }
+    ]
+}
+"@
+
+###
+# Main
+###
+if (-not (Test-Path -Path $installDir)) {
+    Write-Host "Creating $installDir..."
+    New-Item -Path "$installDir" -ItemType Directory | Out-Null
+}
+
+$starttime = Get-Date
+$starttimeString = $($starttime.ToString("yyMMdd-HHmm"))
+$transcriptFile = "$installDir\aksedgedlog-$starttimeString.txt"
+
+Start-Transcript -Path $transcriptFile
+
+Set-ExecutionPolicy Bypass -Scope Process -Force
+# Download the AksEdgeDeploy modules from Azure/AksEdge
+$fork ="angop95"
```

Comment:
We shoudl probably change this to official repo and not  your fork

Reply:
updated